src/components/NurseryWithdrawals/WithdrawalDetails/sections/SpeciesTable.tsx

```diff
@@ -0,0 +1,35 @@
+import { Table, TableColumnType } from '@terraware/web-components';
+import { useEffect, useState } from 'react';
+import strings from 'src/strings';
+import { Delivery } from '../../../../api/types/tracking';
+import { getSpecies } from '../../../../api/species/species';
+import { ServerOrganization } from '../../../../types/Organization';
+
+const columns: TableColumnType[] = [
+  { key: 'species', name: strings.SPECIES, type: 'string' },
+  { key: 'to_plot', name: strings.TO_PLOT, type: 'string' },
+  { key: 'quantity', name: strings.QUANTITY, type: 'string' },
+];
+
+type SpeciesTableSectionProps = {
+  organization: ServerOrganization;
+  delivery?: Delivery;
+};
+
+export default function SpeciesTable({ organization, delivery }: SpeciesTableSectionProps): JSX.Element {
+  const [rowData, setRowData] = useState<{ [p: string]: unknown }[]>([]);
+
+  useEffect(() => {
+    const rows: { [p: string]: unknown }[] = [];
+    delivery?.plantings?.forEach(async (planting) => {
+      rows.push({
+        species: (await getSpecies(planting.speciesId, organization.id.toString()))?.species?.scientificName ?? '',
```

Comment:
We have an array of all the species of the org on the NurseryWithdrawals parent component. We could pass the species array to the withdrawal detail and then to this table and here we could do speciesList.find(species => species.id === planting.speciesId).scientificName, so we don't have to do an api call for each row

Reply:
good call, I'll get that data down to here to avoid those calls 👍